src/core-services/product/queries/products.js

```diff
@@ -0,0 +1,27 @@
+import applyProductFilters from "../utils/applyProductFilters.js";
+
+/**
+ * @name products
+ * @method
+ * @memberof GraphQL/Products
+ * @summary Query the Products collection for a list of products
+ * @param {Object} context - an object containing the per-request state
+ * @param {Object} input - ID of Shop to query against
+ * @returns {Promise<Object>} Products object Promise
+ */
+export default async function products(context, input) {
+  const { checkPermissions, collections } = context;
+  const { Products } = collections;
+  const { ...productFilters } = input;
+
+  // Check the permissions for all shop requested
+  await Promise.all(productFilters.shopIds.map(async (shopId) => (
```

Comment:
Nothing for you to do here, just making a note for myself to update this in the auth pod once this is merged.